covid19_sfbayarea/data/marin_scraper.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+import csv
+import json
+import numpy as np
+from typing import List, Dict, Tuple
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from urllib.parse import unquote_plus
+from datetime import datetime
+import re
+
+from .utils import get_data_model
+
+def get_county_data() -> Dict:
```

Comment:
This needs to be named `get_county`!

```suggestion
def get_county() -> Dict:
```